gr-blocks/python/blocks/qa_moving_average.py

```diff
@@ -87,5 +87,41 @@ def test_02(self):
         # make sure result is close to zero
         self.assertComplexTuplesAlmostEqual(expected_result, dst_data, 1)
 
+    def test_03(self):
+        tb = self.tb
+
+        print "setting up"
+        vlen = 5
+        N = 10*vlen
+        seed = 0
+        data = make_random_float_tuple(N, 2**10)
+        data = [int(d*1000) for d in data]
+        print "making blocks"
```

Comment:
These print statements should go -- they're not really helpful, they look like someone was debugging segfaults, and they're not Py3k compatible.

Reply:
@marcusmueller please take care of this

Reply:
@jmcorgan took care. Please let me know if changeset is OK now; I will then squash commits. By the way, I currently do not like github's PR work flow. Feels like the whole PR-review-update branch-review-… cycle is less clean than the usual "patch via mailing list" approach.

Reply:
Let's not have a discussion on the tooling. It is, however, a bit cleaner if you you `--fixup` commits for easier tracking of updates in the git history.